content/overview/getting-started/_index.en.md

```diff
@@ -5,50 +5,149 @@ weight: 2
 draft: false
 ---
 
-### Global site parameters
+In this guide, we'll be covering the following starting actions for wasmCloud development:
+1. Launching a wasmCloud Host
+1. Starting a wasmCloud Actor
+1. Starting a wasmCloud Capability Provider
+1. Linking an Actor to a Capability Provider
+1. Making requests to an Actor
 
-On top of [Hugo global configuration](https://gohugo.io/overview/configuration/), **Dot** lets you define the following parameters in your `config.toml` (here, values are default).
+### Prerequisites
+In order to follow this guide, you'll need a few things:
+- `wash` (installation covered on [the previous page](../installation))
+- [nats-server](https://docs.nats.io/nats-server/installation) **OR** [docker](https://docs.docker.com/engine/install/), recommended with [docker-compose](https://docs.docker.com/compose/install/)
 
-Note that some of these parameters are explained in details in other sections of this documentation.
+### Starting NATS
+[NATS](https://nats.io/) is a reliable cross-environment messaging system that wasmCloud uses extensively to send messages between the core program components. wasmCloud _can_ be run without using NATS, however this is only recommended in extremely constrained environments, or when NATS is unavailable.
 
-```toml
-# base URL place here
-baseURL = "https://examplesite.com"
-# this is your site title
-title = "Hugo documentation theme"
-# theme should be `godocs`
-theme = "godocs"
-# disable language from here
-disableLanguages = ["fr"] # now franch language is disabled
+In a terminal window, use one of the following options to launch `nats-server` locally. We recommend using the `docker-compose` as it will also launch services that are used in other places in this documentation.
 
+{{% tabs %}}
+   {{% tab "docker-compose" %}}
+    # This file is located in the wash repository under tools/
+    cat <<EOF > docker-compose.yml
+    version: "3"
+    services:
+      registry:
+        image: registry:2
+        ports:
+        - "5000:5000"
+      nats:
+        image: nats:2.1.9
+        ports:
+        - "6222:6222"
+        - "4222:4222"
+        - "8222:8222"
+      redis:
+        image: redis:6.0.9
+        ports:
+        - "6379:6379"   
+    EOF
+    docker-compose up
+   {{% /tab %}}
+   {{% tab "docker" %}}
+    docker run -p 4222:4222 -ti nats:latest
+   {{% /tab %}}
+   {{% tab "nats-server binary" %}}
+    nats-server -a 0.0.0.0 -p 4222
+   {{% /tab %}}
+ {{% /tabs %}}
 
-# add css plugin
-[[params.plugins.css]]
-link = "define plugins path"
+### Running your first wasmCloud Host
+In a separate terminal window, run the following command to launch an interactive wasmCloud REPL environment, including a preconfigured wasmCloud host.

```

Comment:
typical convention leading up to a code block and we use the word/phrase "the following" is to end this sentence with a `:` instead of a `.`. 

I'm not sure which AP style guide that comes from, but it's a thing ;)

Reply:
That makes sense, I believe I did that in other places too but I'll comb through and make that consistent